Dfe.Academies.External.Web.UnitTest/Dfe.Academies.External.Web.UnitTest.csproj

```diff
@@ -24,6 +24,7 @@
 
   <ItemGroup>
     <Folder Include="Models\" />
+    <Folder Include="ViewModels\" />
```

Comment:
Where have these come from, do we need them? I'm sure that csproj files don't need there Includes any more, I think they can go in by accident when merging.